src/slicer.cpp

```diff
@@ -780,6 +780,9 @@ void SlicerLayer::makePolygons(const Mesh* mesh, bool keep_none_closed, bool ext
     polygons.removeDegenerateVerts(); // remove verts connected to overlapping line segments
 
     int xy_offset = mesh->getSettingInMicrons("xy_offset");
+    if(is_initial_layer)
+        xy_offset = mesh->getSettingInMicrons("xy_offset_layer_0");
+
```

Comment:
I expected the initial layer XY offset to add onto the normal xy offset.

The initial layer XY offset is to get rid of elephant feat.

The normal XY offset is to compensate for hole dimensions for example.

If you do a large negative xy_offset it would be weird that you get the elephant feat back again because you didn't also change the xy_offset_layer_0...